### ISSUE TITLE
Membrane (user-) group support is broken

### ISSUE THREAD

**Issue by petri** [collective/dexterity.membrane#17]
- wrong group behavior factory
- no adapter from IMembraneGroup to IGroup

**Comment by petri** [comment]
See [petri-fix-group](https://github.com/collective/dexterity.membrane/tree/petri-fix-group) branch for an attempt to improve the membrane group support.

**Comment by gforcada** [comment]
@petri you can create pull requests from your branch so it is easier to track the changes and review them.

**Comment by petri** [comment]
Yes. Just wanted to tidy up a bit first. You can get a nice review view without PR, too, such as: https://github.com/collective/dexterity.membrane/compare/petri-fix-group

**Comment by petri** [comment]
@gforcada , any others, could you review the PR?
